### PR TITLE
BD-2469: Feature Flag experiments

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-2469**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_developer_guide/platform_wide/feature_flags/experiments.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+nav_title: Feature Flag Experiments
+article_title: Feature Flag Experiments
+page_order: 40
+description: " "
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/platform_wide/feature_flags/experiments.md:5]:
```suggestion
description: "Feature flag experiments let you A/B test changes to your applications to optimize conversion rates."
```

File: _docs/_developer_guide/platform_wide/feature_flags/experiments.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+```
+
+## Step 1: Create an experiment
+
+1. Go to **Messaging** > **Campaigns** and click **+ Create Campaign**.
+2. Select **Feature Flag Experiment**.
+3. Name your campaign something clear and meaningful.
+4. Add [Teams][1] and [Tags][2] as needed.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/feature_flags/experiments.md:40]:
```suggestion
```
we don't support it yet

File: _docs/_developer_guide/platform_wide/feature_flags/experiments.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - Tags make your campaigns easier to find and build reports out of. For example, when using the [Report Builder][3], you can filter by particular tags.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/feature_flags/experiments.md:41]:
```suggestion
```

File: _docs/_developer_guide/platform_wide/feature_flags/experiments.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Next, create variations. For each variant, choose the feature flag you want to turn on or off and review the assigned properties.
+
+To test the impact of your feature, use variants to split traffic into two or more groups. One variant should be considered your control group—this variant should have the feature flag turned off.
+
+### Overwriting properties
+
+While you specified default properties when you originally set up your feature flag, you can choose to overwrite those values for users who receive a specific campaign variant.
+
+![][image1]{: style="max-width:80%"}
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/feature_flags/experiments.md:53]: is this image from canvas? it's missing the on/off switch.

Comment (bre-fitzgerald) [reply _docs/_developer_guide/platform_wide/feature_flags/experiments.md:53]: Ah whoops yes, let me recreate this in a campaign

File: _docs/_developer_guide/platform_wide/feature_flags/experiments.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+To edit, add, or remove additional default properties, edit the feature flag itself from **Messaging** > **Feature Flags**.
+
+## Step 3: Choose users to target
+
+Next, you need to [target users][4] by choosing segments or filters to narrow down your audience. Segment membership is calculated when feature flags are refreshed for a given user.
+
+{% alert note %}
+Your target audience will be eligible for the feature flag immediately, when your app refreshes feature flags, or when a new session is started.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/feature_flags/experiments.md:60]:
```suggestion
Your target audience will be eligible for the feature flag as soon as you save a rollout greater than 0%. Changes are made available once your app refreshes feature flags, or when a new session is started.
```

**Review by davidbielik — APPROVED** [review]
lgtm - left a few comments

File: _docs/_developer_guide/platform_wide/feature_flags/experiments.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+# Creating a feature flag experiment
+
+> Feature flag experiments let you A/B test changes to your applications to optimize conversion rates. Marketers can use feature flags to determine whether a new feature positively or negatively impacts conversion rates, or which set of feature flag properties is most optimal.
+
+## Prerequisites
+
+Before you can track user data in the experiment, your app needs to record when a user interacts with a feature flag. This is called a feature flag impression. Make sure to log a feature flag impression whenever a user sees or could have seen the feature you're testing, even if they're in the control group.
+
+To learn more, see [Logging a feature flag impression][6].
```
Comment (rachel-feinberg) [inline _docs/_developer_guide/platform_wide/feature_flags/experiments.md:22]: Just double-checking if this should be "Creating feature flags", or something like the below.

```suggestion
To learn more, see [Logging a feature flag impression][6].
```
To learn more about logging feature flag impressions, see [Creating feature flags][6].

File: _docs/_developer_guide/platform_wide/feature_flags/experiments.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+## Step 2: Add experiment variants
+
+Next, create variations. For each variant, choose the feature flag you want to turn on or off and review the assigned properties.
+
+To test the impact of your feature, use variants to split traffic into two or more groups. One variant should be considered your control group—this variant should have the feature flag turned off.
+
+### Overwriting properties
+
+While you specified default properties when you originally set up your feature flag, you can choose to overwrite those values for users who receive a specific campaign variant.
```
Comment (rachel-feinberg) [inline _docs/_developer_guide/platform_wide/feature_flags/experiments.md:49]:
```suggestion
Though you specified default properties when you originally set up your feature flag, you can choose to overwrite those values for users who receive a specific campaign variant.
```

**Review by rachel-feinberg — APPROVED** [review]
Lgtm! Left two small comments

**Pull request merged**